gradle/dependencies.gradle

```diff
@@ -158,9 +158,6 @@ versions += [
   slf4j: "1.7.36",
   snappy: "1.1.10.5",
   spotbugs: "4.7.3",
-  // New version of Swagger 2.2.14 requires minimum JDK 11.
```

Comment:
Can we keep this comment in gradle.properties?
I created https://issues.apache.org/jira/browse/KAFKA-15549 to remember us we will be able to bump it once support for Java 8 is removed.